### ISSUE TITLE
Fix .Rd image height attribute

### ISSUE THREAD

**Issue by riccardoporreca** [miraisolutions/rTRNG#25]
From CRAN maintainer's email:

> [...]
> Rd files which when converted to HTML have <img> entries with
invalid height or width attributes.  E.g.,
> ....Rd:   \figure{....png}{options: width="50px" alt="..."}
> 
> The current HTML standard says at
> <https://html.spec.whatwg.org/#attr-dim-width> that
>
>  The width and height attributes on img, iframe, embed, object, video,
  source when the parent is a picture element and, when their type
  attribute is in the Image Button state, input elements may be
  specified to give the dimensions of the visual content of the element
  (the width and height respectively, relative to the nominal direction
  of the output medium), in CSS pixels. The attributes, if specified,
  must have values that are valid non-negative integers.
>
> so the above should use
> options: width="50"
>
> Problems found in your package Rd files
>  rTRNG::rTRNG-package.Rd: Warning: <img> attribute "height" has invalid value "150px"
>
>  **Please fix before 2022-03-16 to safely retain your package on CRAN.**

**Comment by riccardoporreca** [comment]
Detail about reproducing the issue and motivation from a follow-up email

> use HTML Tidy <https://www.html-tidy.org/> for validation.  With this
installed an on the path, you can do something like
>
>  out <- tempfile()
  tools::Rd2HTML("/data/rsync/PKGS/rTRNG/man/rTRNG-package.Rd", out)
  system2("tidy", c("-qe", out))
>
> which will give
>
>  line 22 column 4 - Warning: <img> attribute "height" has invalid value "150px"
>
> Rd2HTML validation will soon get integrated into the CRAN submission
checks: but for now it will skip Rd files auto-generated by roxygen2 (so
yours would be missed).

**Comment by riccardoporreca** [comment]
Addressed in PR #28.

As advised above, the whole documentation has been validated locally using https://www.html-tidy.org/:

```r
out <- tempfile()
for (rd_file in list.files("man", "*[.]Rd$", full.names = TRUE)) {
  message(rd_file)
  tools::Rd2HTML(rd_file, out)
  system2("tidy", c("-qe", out))
}
```
